src/app/views/users/_user.html.erb

```diff
@@ -11,8 +11,20 @@
           <%= link_to user.name, user_functions_path(user) %>
         </p>
         <p class="subtitle is-6">
-          @<%= user.username %>
+          <%= link_to user_github_account_url(user),  target: "_blank" do %>
+            @<%= user.username %>
+          <% end %>
         </p>
+        
+        <% unless current_user.nil? || user.id == current_user.id %> 
```

Comment:
rails compares models by primary key , and current_user is truthy if it's not nil
```suggestion
        <% if current_user && user != current_user %> 
```